src/components/GameCard.tsx

```diff
@@ -126,16 +142,22 @@ export const GameCard: React.FC<GameCardProps> = ({
             </div>
           )}
         </div>
+
+        {enableSteamWidgets && !isIframeLoaded && <WidgetSkeleton />}
         {enableSteamWidgets && steamAppId && (
-          <div className="relative w-full mt-4">
+          <div className="w-full mt-4">
             <iframe
               src={`https://store.steampowered.com/widget/${steamAppId}/`}
-              width="646"
-              height="190"
+              className="w-full h-[187px]"
+              onLoad={() => setIsIframeLoaded(true)}
             ></iframe>
           </div>
         )}
       </CardContent>
     </Card>
   );
 };
+
+const WidgetSkeleton = () => (
+  <div className="w-full h-[187px] mt-4 bg-gray-200 dark:bg-gray-800 rounded-md animate-pulse" />
+);
```

Comment:
The ones without steam widget show like this: (with the skeleton loading)
it should have a steam id

<img width="690" alt="Screenshot 2024-12-19 at 09 41 22" src="https://github.com/user-attachments/assets/584e0041-ecfa-4ffe-81c4-83e975fc710f" />

Reply:
Now is hidden when no steam id